packages/web/tailwind.config.js

```diff
@@ -142,13 +142,17 @@ module.exports = {
       h5: 600,
       h6: 600,
       subtitle1: 600,
-      subtitle2: 600,
-      body1: 500,
-      body2: 500,
-      button: 700,
-      caption: 500,
+      subtitle2: 400,
+      body1: 400,
+      body2: 400,
+      button: 600,
+      caption: 400,
       overline: 400,
     },
+// body1: 500 -> 400
```

Comment:
Could we remove these comments? We can see preview values in git history.

Reply:
yeah lol, I'm kinda OCD about keeping notes. I'll update